src/app/beer_garden/requests.py

```diff
@@ -729,20 +734,18 @@ def process_request(
             wait_timeout = None
 
         try:
-            completed = request_map[request_id].wait(timeout=wait_timeout)
+            completed = request_map[request.id].wait(timeout=wait_timeout)
             if not completed:
-                raise TimeoutError(f"Timeout exceeded for request {request_id}")
+                raise TimeoutError(f"Timeout exceeded for request {request.id}")
 
-            request.reload()
+            request = reload(request)
```

Comment:
another good reason to namespace it as `db.reload(`

Reply:
See 453a765